en/guide/toolchain.md

```diff
@@ -1,9 +1,10 @@
 # Building C++ Apps
 
-Dronecode SDK C++ apps are written in standard C++ (c++11) and can be built using your preferred build system, compiler and linker toolchain. 
-The only requirement is that the build system must be able to locate the SDK C++ headers and libraries (installed as described [here](../contributing/build.md#install-artifacts)).
+The Dronecode SDK C++ library is written in C++11 and exposes C++11 interfaces such as `std::function`. Therefore, applications using the library need to be C++11 or later as well.
 
-The SDK itself uses the [cmake](https://cmake.org/) build system, and we recommend that you do too. 
+Applications can be built using your preferred build system, compiler and linker toolchain. The only requirement is that the build system must be able to locate the SDK C++ headers and libraries (installed as described [here](../contributing/build.md#install-artifacts)).
```

Comment:
I would remove `(installed as described [here](../contributing/build.md#install-artifacts)).` - later on we're going to update this to build files installed via deb or other method.

Reply:
But as long as we don't have it we need to leave that in, I'd think.

Reply:
Fair enough. Let's plan to revisit when we have a new release.